modules/ROOT/partials/integrations/angular-quick-start.adoc

```diff
@@ -174,7 +169,7 @@ endif::[]
 
 == Deploying the application to a HTTP server
 
-The application will require further configuration before it can be deployed to a production environment. For information on configuring the application for deployment, see: https://angular.io/guide/build[Angular Docs - Building and serving Angular apps] or https://angular.io/guide/deployment[Angular Docs - Deployment].
+The application will require further configuration before it can be deployed to a production environment. For information on configuring the application for deployment, see: https://angular.dev/tools/cli/build[Angular Docs - Building Angular Apps] or https://angular.dev/tools/cli/deployment[Angular Docs - Deployment].
```

Comment:
```suggestion
The application will require further configuration before it can be deployed to a production environment. For information on configuring the application for deployment, see: link:https://angular.dev/tools/cli/build[Angular Docs - Building Angular Apps] or link:https://angular.dev/tools/cli/deployment[Angular Docs - Deployment].
```